### PR TITLE
Add update crd to update-codegen process

### PR DESCRIPTION
**Pull request by cpanato**
# Changes
- update crds as part of the update-codegen process

/kind cleanup

**Release Note**

```release-note
update crds as part of the update-codegen process
```

**Docs**

```docs

```

/assign @mattmoor

### CODE REVIEW

File: hack/update-codegen.sh [hunk 1]
```diff
@@ -44,6 +44,16 @@ ${KNATIVE_CODEGEN_PKG}/hack/generate-knative.sh "injection" \
...
   --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
 
+group "Update CRD Schema"
+
+go run $(dirname $0)/../cmd/schema/ dump SimpleDeployment \
+  | yq eval-all --header-preprocess=false --inplace 'select(fileIndex == 0).spec.versions[0].schema.openAPIV3Schema = select(fileIndex == 1) | select(fileIndex == 0)' \
+  $(dirname $0)/../config/300-simpledeployment.yaml -
+
+go run $(dirname $0)/../cmd/schema/ dump AddressableService \
+  | yq eval-all --header-preprocess=false --inplace 'select(fileIndex == 0).spec.versions[0].schema.openAPIV3Schema = select(fileIndex == 1) | select(fileIndex == 0)' \
```
Comment (dprotaso) [inline hack/update-codegen.sh:54]: what's the select for?

File: third_party/VENDOR-LICENSE/LICENSE [hunk 2]
```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
```
Comment (dprotaso) [inline third_party/VENDOR-LICENSE/LICENSE:1]: I think these are not needed. Can you update your go-licenses

```
go install github.com/google/go-licenses@master
```

File: hack/update-codegen.sh (hunk 1 above)
Comment (dprotaso) [reply hack/update-codegen.sh:54]: also `yq` isn't in our build image 

we do have a `run_go_tool` helper

you can see how to use it here

https://github.com/knative/serving/blob/80be89b59e2dd2ed5c351cec9d729fa1b4e450a3/test/e2e-common.sh#L568-L570

Comment (cpanato) [reply hack/update-codegen.sh:54]: it applies all the changes that were generated by running the `go run $(dirname $0)/../cmd/schema/ dump SimpleDeployment` (for example) in inside the `openAPIV3Schema` which is the data that is dumped from the schema dump

Comment (cpanato) [reply hack/update-codegen.sh]: Thanks for your review and feedback. I added the tool following the example you gave

File: third_party/VENDOR-LICENSE/LICENSE (hunk 2 above)
Comment (cpanato) [reply third_party/VENDOR-LICENSE/LICENSE]: dropped

**Pull request merged**